.github/workflows/test.yml

```diff
@@ -8,7 +8,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v2
-    - uses: cachix/install-nix-action@v12
+    - uses: cachix/install-nix-action@v13
+      with:
+        # We need a nix version with NixOS/nix#3564 to be able to use
+        # `--max-jobs`, so using NixOS/nix@323e545. See 8365e73 for the problem
+        # with `--max-jobs`. Install URL is obtained from CI logs for the nix
+        # commit.
+        install_url: https://nixos-nix-install-tests.cachix.org/serve/i6laym9jw3wg9mw6ncyrk6gjx4l34vvx/install
```

Comment:
where does the URL come from? would be great to add a note

Reply:
Done.